apps/rehabstod/src/pages/CurrentSickLeaves/components/TableBodyRows.tsx

```diff
@@ -146,7 +150,7 @@ export function TableBodyRows({
               tabIndex={0}
               onKeyDown={({ code, currentTarget }) => {
                 if (['Enter', 'Space'].includes(code)) {
-                  navigateToPatient(sickLeave.encryptedPatientId)
+                  navigateToPatient(sickLeave)
```

Comment:
Since we are only using the attributes encryptedPatientId and rekoStatus, maybe we should change the method signature and send these values into the method, instead of the whole sickLeave object?

Reply:
I think it's better to leave it generic in case we want to send this as a prop later then the function can be written generic and not so specific.